src/codemagic/models/certificate.py

```diff
@@ -52,6 +52,12 @@ def from_ans1(cls, asn1: AnyStr) -> Certificate:
         x509_certificate = cls._get_x509_certificate(asn1, crypto.FILETYPE_ASN1)
         return Certificate(x509_certificate)
 
+    @classmethod
+    def from_p12(cls, p12: AnyStr, password: Optional[AnyStr] = None) -> Certificate:
```

Comment:
As PKCS 12 is archive file format and contains always binary data, it doesn't make sense to allow `AnyStr` as `p12` input value. 
```suggestion
    def from_p12(cls, p12: bytes, password: Optional[AnyStr] = None) -> Certificate:
```